src/ipyvizzustory/storylib/story.py

```diff
@@ -252,10 +275,10 @@ def get_width_height_in_pixels(self) -> Tuple[int, int]:
         if self.aspect_ratio is None:
             if not StorySize.is_pixel(self.height):
                 raise ValueError("height should be in pixels")
-            return (int(self.width[:-2]), int(self.height[:-2]))  # type: ignore
+            return (int(float(self.width[:-2])), int(float(self.height[:-2])))  # type: ignore
         if not isinstance(self.aspect_ratio, (float, int)):
             raise ValueError("aspect_ratio should be a float")
-        _width = int(self.width[:-2])  # type: ignore
+        _width = int(float(self.width[:-2]))  # type: ignore
         _height = int(_width / self.aspect_ratio)
```

Comment:
Width can be float which is different when it casted to int in variable `_width`.
I think here it need to be use width's float value and not int

Reply:
yes, but in that specific case we need to get the width and height as int (this method only used in specific envs if using the play)

Reply:
fixed in [04aa25d](https://github.com/vizzuhq/ipyvizzu-story/pull/75/commits/04aa25d19c13d57b04211bea75b188ca143f8be1)